Test/scripts/ci-integration.sh

```diff
@@ -4,6 +4,11 @@ set -e
 set -u
 set -x
 
+npm cache clean -f
+npm install -g n
+sudo npm install -g n
+sudo n 14.17.3
```

Comment:
For future portability, should be env variable this and inject it in circle?

Reply:
We are considering this a hotfix. The long time solution is to update the image. I will create task for this.